pkg/api/controllers/transaction_controller_test.go

```diff
@@ -503,6 +503,134 @@ func TestGetTransactions(t *testing.T) {
 	}))
 }
 
+type transaction struct {
+	core.TransactionData
```

Comment:
You can embed core.Transaction directly instead of redefine all fields.
The outer fields 'PreCommitVolumes' and 'PostCommitVolumes' will override inner fields.

Reply:
Yes thanks! :)